token/client/src/token.rs

```diff
@@ -2701,4 +2701,23 @@ where
         ));
         self.process_ixs(&instructions, signing_keypairs).await
     }
+
+    /// Update the token-metadata authority in a mint
+    pub async fn update_authority_in_token_metadata<S: Signers>(
```

Comment:
nit: Does this function name sound a little strange?

I guess no stranger than `update_token_metadata_update_authority`, but the latter sounds a bit more like what folks are used to.

Lmk what you think

Reply:
I wasn't sure for these. I was thinking it either goes "do_thing_in_token_metadata" or "token_metadata_do_thing", so for the second one, we'd have:

* `token_metadata_initialize`
* `token_metadata_update_field`
* `token_metadata_emit`
* `token_metadata_update_authority`
* `token_metadata_remove_key`

Maybe those are better now that I write them out. What do you think?

Reply:
Not a hill I'd die on but I will say those do mimic the Rust full path quite nicely, ie:

```rust
token_metadata::instruction::Initialize,
token_metadata::instruction::UpdateAuthority,
...
```

It only just jumped out at me with update authority (not the prior two PRs)

Reply:
works for me!